crm_task/views/project_task.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    
+    <!-- FORM -->
+    <record id="project_task_form2" model="ir.ui.view">
+        <field name="name">project.task.form.2</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="hr_timesheet.view_task_form2_inherited"/>
+        <field name="arch" type="xml">
+
+            <xpath expr="//sheet//field[@name='project_id']" position="before">
+                <field name="lead_id"/>
+            </xpath>
+
+        </field>
+    </record>
+
+    <!-- TREE -->
+    <record id="project_task_tree2" model="ir.ui.view">
+        <field name="name">project.task.tree.2</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_tree2"/>
+        <field name="arch" type="xml">
+
+            <xpath expr="//tree//field[@name='name']" position="before">
+                <field name="lead_id"/>
+            </xpath>
+
+        </field>
+    </record>
+
+    <!-- SEARCH -->
+    <record id="view_task_search_form" model="ir.ui.view">
+        <field name="name">Search</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_search_form"/>
+        <field name="arch" type="xml">
+            
+            <field name="name" position="after">
+                <field name="lead_id" string="Opportunity" filter_domain="[('lead_id','ilike', self)]"/>
+            </field>
+
+        </field>
+    </record> 
+
+    <!-- ACTION -->
+    <record id="crm_task_to_opportunity" model="ir.actions.act_window">
```

Comment:
Why do you create a new action? Can't you use one of the existing?

Reply:
Okay. A little more, I would move the "opportunity" field down to "Assigned to" in the task view.
![Captura de pantalla 2020-07-14 a las 12 33 25](https://user-images.githubusercontent.com/22913794/87416211-aa573c00-c5ce-11ea-8a54-664408182649.png)